src/main/java/org/fcrepo/spec/testsuite/AbstractTest.java

```diff
@@ -205,6 +205,13 @@ protected Response createDirectContainerUnverifed(final String uri, final String
         return doPostUnverified(uri, headers, body);
     }
 
+    protected Response createIndirectContainerUnverifed(final String uri, final String body) {
```

Comment:
It is not clear that a `body` parameter is needed here, since the only usage of this method passes in `""`. Maybe create a dummy body within this method?

Reply:
👍